torch/_torch_docs.py

```diff
@@ -5821,6 +5821,64 @@ def merge_dicts(*dicts):
              [ 0,  1]]])
 """.format(**common_args))
 
+add_docstr(torch.fliplr,
+           r"""
+fliplr(input) -> Tensor
+
+Flip array in the left/right direction.
```

Comment:
I know this is what the NumPy documentation says, which is a smart start, but I'd like to clarify the behavior for what happens when the tensor has more than 2 dimensions. In that case is a "row" what you'd expect?

Reply:
Oh! And I almost forgot: we should clarify that PyTorch returns a new tensor:

"Flips array in the left/right direction, returning a new tensor."